File: CemQuintas.API/Models/Produtividade.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CemQuintas.API.Models
{
    [Serializable]
    public class Produtividade
    {
        private string _login;
        private int _quantidade;

        public string Login { get => _login; set => _login = value; }
        public int Quantidade { get => _quantidade; set => _quantidade = value; }
    }
}

File: CemQuintas/src/app/Models/produtividade.ts
export class Produtividade {
  private _login: string;
  private _quantidade: number;

  public get Login(): string {
    return this._login;
  }
  public set Login(value: string) {
    this._login = value;
  }
  public get Quantidade(): number {
    return this._quantidade;
  }
  public set Quantidade(value: number) {
    this._quantidade = value;
  }
}
